isthmus/src/test/java/io/substrait/isthmus/CalciteLiteralTest.java

```diff
@@ -215,6 +215,29 @@ void tIntervalMillisecond() {
     bitest(intervalDaySecondExpr, intervalDaySecond);
   }
 
+  @Test
+  void tIntervalDay() {
+    // Calcite always uses milliseconds
+    BigDecimal bd = new BigDecimal(TimeUnit.DAYS.toMillis(5));
```

Comment:
maybe use some value that's days + seconds + millis rather than just days?

Reply:
This test is specifically testing day intervals since that is the case I hit the issue. It wouldn't be a good test of day intervals if it actually tested other time units. I am happy to add additional test cases though. There is already a day/decond test immediately above this one in the file. What other cases do you think it would be valuable to cover? Just day + second + millis?